### PR TITLE
closes #115; closes #118

### LINKED ISSUE #115: including CI columns in output dataframes as a fraction
As decided in easystats/effectsize#280 (comment)

```r
library(correlation)
results <- correlation(iris)

as.data.frame(results)
#>     Parameter1   Parameter2          r     CI_low     CI_high         t  df
#> 1 Sepal.Length  Sepal.Width -0.1175698 -0.2726932  0.04351158 -1.440287 148
#> 2 Sepal.Length Petal.Length  0.8717538  0.8270363  0.90550805 21.646019 148
#> 3 Sepal.Length  Petal.Width  0.8179411  0.7568971  0.86483606 17.296454 148
#> 4  Sepal.Width Petal.Length -0.4284401 -0.5508771 -0.28794993 -5.768449 148
#> 5  Sepal.Width  Petal.Width -0.3661259 -0.4972130 -0.21869663 -4.786461 148
#> 6 Petal.Length  Petal.Width  0.9628654  0.9490525  0.97298532 43.387237 148
#>              p  Method n_Obs
#> 1 1.518983e-01 Pearson   150
#> 2 5.193337e-47 Pearson   150
#> 3 9.301992e-37 Pearson   150
#> 4 1.353994e-07 Pearson   150
#> 5 8.146457e-06 Pearson   150
#> 6 2.805002e-85 Pearson   150
```

### LINKED ISSUE #118: `df` column should be renamed to `df_error`
To be internally consistent in the `easystats` ecosystem:

``` r
x <- c(44.4, 45.9, 41.9, 53.3, 44.7, 44.1, 50.7, 45.2, 60.1)
y <- c( 2.6,  3.1,  2.5,  5.0,  3.6,  4.0,  5.2,  2.8,  3.8)
df <- data.frame(x = x, y = y)
  
as.data.frame(parameters::model_parameters(cor.test(x, y)))
#>   Parameter1 Parameter2         r        t df_error         p     CI_low
#> 1          x          y 0.5711816 1.841083        7 0.1081731 -0.1497426
#>     CI_high                               Method
#> 1 0.8955795 Pearson's product-moment correlation

as.data.frame(correlation::correlation(df))
#>   Parameter1 Parameter2         r     CI_low   CI_high        t df         p
#> 1          x          y 0.5711816 -0.1497426 0.8955795 1.841083  7 0.1081731
#>    Method n_Obs
#> 1 Pearson     9
```

<sup>Created on 2021-02-02 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
Hmm, vignettes are not being built, so I have messed something up.

**Comment by DominiqueMakowski** [comment]
It fails for multilevel for some obscure reason, let me see

``` r
correlation::correlation(iris, multilevel=TRUE)
#> Error in if (det(m) < tol) {: missing value where TRUE/FALSE needed

6.
.invert_matrix(m, tol = tol) at cor_to_pcor.R#242
5.
.pcor_to_cor(.get_cor(summary(pcor, redundant = TRUE), cov = NULL)) at cor_to_pcor.R#118
4.
.cor_to_pcor_easycorrelation(pcor = pcor, tol = tol) at cor_to_pcor.R#97
3.
pcor_to_cor.easycorrelation(pcor = out) at cor_to_pcor.R#76
2.
pcor_to_cor(pcor = out) at correlation.R#301
1.
correlation(iris, multilevel = TRUE) 

```

<sup>Created on 2021-02-03 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by DominiqueMakowski** [comment]
Weird, it *seems* it fails because the matrix returned by the following has NANs in its diagonal, but I don't think there was any change that introduced that... lemme see on the master branch

``` r
correlation:::.get_cor(summary(correlation::correlation(iris), redundant = TRUE))
#>              Sepal.Length Sepal.Width Petal.Length Petal.Width
#> Sepal.Length           NA  -0.1175698    0.8717538   0.8179411
#> Sepal.Width    -0.1175698          NA   -0.4284401  -0.3661259
#> Petal.Length    0.8717538  -0.4284401           NA   0.9628654
#> Petal.Width     0.8179411  -0.3661259    0.9628654          NA
```

<sup>Created on 2021-02-03 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by DominiqueMakowski** [comment]
On master it returned a regular matrix indeed:

``` r
correlation:::.get_cor(summary(correlation::correlation(iris), redundant = TRUE))
#>              Sepal.Length Sepal.Width Petal.Length Petal.Width
#> Sepal.Length    1.0000000  -0.1175698    0.8717538   0.8179411
#> Sepal.Width    -0.1175698   1.0000000   -0.4284401  -0.3661259
#> Petal.Length    0.8717538  -0.4284401    1.0000000   0.9628654
#> Petal.Width     0.8179411  -0.3661259    0.9628654   1.0000000
```

<sup>Created on 2021-02-03 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>


Mmmh okay I'll start readding your changes step by step, so the 3 main things are : 

- [ ] Add CI as a column (as a fraction)
- [ ] Rename df -> df_error
- [ ] Reorder df_error to match parameters

File: R/utils_bootstrapping.R [hunk 1]
```diff
@@ -45,7 +45,7 @@
     # bootstrap values or quantiles
     if (method == "normal") {
       # get bootstrap standard error
-      bootse <- stats::qt((1 + ci.lvl) / 2, df = length(stats::na.omit(x)) - 1) * stats::sd(x, na.rm = T)
+      bootse <- stats::qt((1 + ci.lvl) / 2, df_error = length(stats::na.omit(x)) - 1) * stats::sd(x, na.rm = T)
```
Comment (DominiqueMakowski) [inline R/utils_bootstrapping.R:48]: and this probably should stayed 'df' 😬

**Pull request merged**